app/src/main/java/com/keylesspalace/tusky/StatusListActivity.kt

```diff
@@ -262,6 +276,24 @@ class StatusListActivity : BottomSheetActivity(), HasAndroidInjector {
                     }
                 }
             )
+
+            if (filterCreateSuccess) {
+                updateTagMuteState(true)
```

Comment:
Can this be moved outside the `if` block, and just be `updateTagMuteState(filterCreateSuccess)` ?

Reply:
It was "inside" before.

Moving it outside would set it to a different (equally not-meaningful) state when there is an error.